lib/scoped_search/definition.rb

```diff
@@ -334,6 +334,12 @@ def register_named_scope! # :nodoc
 
         search_scope
       end
+
+      # This should get called only when the relative specifiers (1 day ago and
+      # so on), this code path is only viable when Rails are available
```

Comment:
It looks like this lib can be used out of Rails app, so should this method be more secure? Something like https://github.com/wvanbergen/scoped_search/blob/master/lib/scoped_search/definition.rb#L291.

Reply:
Done, it should now act as an identity function if the railsy bits are unavailable